### ISSUE TITLE
ShapeShift's Colony on xDai isn't loading for some members. Possible issue with subgraph?

### ISSUE THREAD

**Issue by LPX55** [JoinColony/colonyDapp#2732]
## Steps to reproduce

1. Load the page as usual.

## Expected behaviour

* Example: Page loads... slow, but eventually it loads.

## Actual behaviour

* Example: Homepage doesn't load. Events and extensions and certain other meta data loads fine at times.

- [Network logs](https://i.imgur.com/XXgTMFH.png)
- [What I've been seeing the past day](https://i.imgur.com/11vGZ2G.png)

**Comment by rdig** [comment]
Yep, this definitely looks like the dapp failed to fetch data from the subgraph

**Comment by LPX55** [comment]
Seems to be fetching the data fine now. Not sure if this is something worth taking a deeper look into, as it has happened before as well. Depending on your answer, I will either leave it open or close this.
